client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/BusinessContactForm/BusinessContactForm.tsx

```diff
@@ -30,22 +30,37 @@ const BusinessContactForm : React.FC = () : JSX.Element => {
                     <FormInput fieldName={businessContactFirstNameField}>
                         <CircleTextField
                             value={contactPersonFirstName}
-                            onChange={event => onChange(event, InteractionEventDialogFields.CONTACT_PERSON_FIRST_NAME)}/>
+                            onChange={event => onChange(event.target.value, InteractionEventDialogFields.CONTACT_PERSON_FIRST_NAME)}/>
                     </FormInput>
                 </Grid>
                 <Grid item xs={4}>
                     <FormInput fieldName={businessContactLastNameField}>
                         <CircleTextField
                             value={contactPersonLastName}
-                            onChange={event => onChange(event, InteractionEventDialogFields.CONTACT_PERSON_LAST_NAME)}/>
+                            onChange={event => onChange(event.target.value, InteractionEventDialogFields.CONTACT_PERSON_LAST_NAME)}/>
                     </FormInput>
                 </Grid>
                 <Grid item xs={4}>
                     <FormInput fieldName={businessContactNumField}>
                         <PhoneNumberTextField
                             id={'businessContactedPersonPhone'}
-                            value={contactPersonPhoneNumber}
-                            onChange={event => onChange(event, InteractionEventDialogFields.CONTACT_PERSON_PHONE_NUMBER)}
+                            value={contactPersonPhoneNumber?.number}
+                            isValid={ctxt.interactionEventDialogData.contactPersonPhoneNumber?.isValid as boolean}
+                            setIsValid={isValid => onChange(
+                                {
```

Comment:
spread ctxt.interactionEventDialogData.contactPersonPhoneNumber and just change isValid